### DISCUSSION TITLE
B站付费课堂视频如何下载

### DISCUSSION THREAD

**Discussion by HeyWake** [leiurayer/downkyi#1023] [Q&A]
B站付费课程课堂如何下载呀？只有先买课有了链接才能解析下载吗？

**Comment by momo2008** [comment]
肯定要先购买呀



发自我的iPhone


------------------ 原始邮件 ------------------
发件人: HeyWake ***@***.***&gt;
发送时间: 2024年1月28日 19:54
收件人: leiurayer/downkyi ***@***.***&gt;
抄送: Subscribed ***@***.***&gt;
主题: Re: [leiurayer/downkyi] B站付费课堂视频如何下载 (Discussion #1023)





 
B站付费课程课堂如何下载呀？只有先买课有了链接才能解析下载吗？
